### PR TITLE
update fix

### PR DESCRIPTION
**Pull request by Caesarsage**

### CODE REVIEW

File: server/public/userTestUpdate.html [hunk 1]
```diff
@@ -54,7 +54,7 @@ <h3>Chess Room user details</h3>
       console.log(organization.value.toString());
       console.log(wins.value.toString());
       console.log(id.value.toString());
-      const url = `https://chess.zuri.chat/api/dbUpdate/${id.value}`
+      const url = `http://localhost:5050/api/dbUpdate/${id.value}`
```
Comment (eni4sure) [inline server/public/userTestUpdate.html:57]: fixxx

Comment (Caesarsage) [reply server/public/userTestUpdate.html:57]: > fixxx thiss

done - I am using the localhost

Comment (eni4sure) [reply server/public/userTestUpdate.html:57]: You cannot use localhost on the live server boss 🙂

**Review by eni4sure — CHANGES_REQUESTED** [review]

**Comment by eni4sure** [comment]
Opened too long

**Pull request closed without merging**